src/Uno.UWP/Buffer/DefaultArrayPool.cs

```diff
@@ -129,7 +127,7 @@ public T[] Rent(int minimumLength)
 			{
 				// No need for events with the empty array.  Our pool is effectively infinite
 				// and we'll never allocate for rents and never store for returns.
-				return s_emptyArray ?? (s_emptyArray = Array.Empty<T>());
+				return Array.Empty<T>();
```

Comment:
This used to be a zero-length array allocation, which I updated to `Array.Empty<T>` in an earlier PR.

The field was there to reduce allocations and have an effect similar to `Array.Empty<T>`. But since we have it now, we can use it directly.